vendor/github.com/containerd/ttrpc/client.go

```diff
@@ -135,7 +137,7 @@ func (c *Client) Call(ctx context.Context, service, method string, req, resp int
 	}
 
 	if cresp.Status == nil {
-		return errors.New("no status provided on response")
+		return status.ErrorProto(&spb.Status{Code: int32(codes.OK)})
```

Comment:
Quick hack to see if it's indeed in situations where there was no error (thus `status` being `nil`)

Reply:
Ah, booh; I think CI overwrites local changes in the vendor step, because I still get the same error

```
INFO[0000] start to pull seed image                     
1404--- FAIL: TestClientTTRPC_Reconnect (0.00s)
1405    client_ttrpc_test.go:56: assertion failed: error is not nil: no status provided on response
```